packages/commerce/backend/src/open-payments/service.ts

```diff
@@ -155,9 +153,85 @@ export class OpenPayments implements IOpenPayments {
       throw new InternalServerError()
     })
 
+    await order.$query().patch({
+      quoteId: quote.id,
+      paymentPointerUrl: customerPaymentPointer.id,
+      continueToken: outgoingPaymentGrant.continue.access_token.value,
+      continueUri: outgoingPaymentGrant.continue.uri
+        .replace('localhost', 'rafiki-auth')
```

Comment:
is this going to work on deployed version? should here be some env variables instead?

Reply:
I'll add a check to only replace it in a dev environment. 👍